src/app.ts

```diff
@@ -72,12 +74,14 @@ class App {
         return res.send(swaggerUi.generateHTML(swaggerJSONDoc, swagger_options))
       }
     )
-    this.express.use(Authentication.handleError)
-    this.express.use(Authentication.accessControl)
+    this.express.use(auth.handleError)
+    this.express.use(async (req, res, next) => await auth.accessControl(req, res, next))
   }
 
   private routes() {
     const app = this.express
+    const rawMiddleware = express.raw({ type: 'application/octet-stream' })
```

Comment:
Please move this middeware to the middleware() function. we need not declare a variable and use it in the route.